doc/liblouis.texi

```diff
@@ -1612,11 +1610,21 @@ For examples see @file{el.ctb}.
 
 @table @code
 
-@opcode{class, name characters}
-Define a new character class. The name operand must contain only
-letters (a-z and A-Z). The letters may be upper or lower-case but the
-case matters. The characters operand must be specified as a string. A
-character class may not be used until it has been defined.
+@opcode{attribute, name characters}
+Add characters to a character class. The class may be one of the
+predefined classes listed above, a user-defined class previously
+created with this opcode, or a new one. The name operand must either
+contain only letters (a-z and A-Z, case matters), or it must be a
+number between 0 and 7. The characters operand must be specified as a
```

Comment:
I really do not like the fact that you can have character classes named `1`. Maybe we should not even mention this in the manual. Can't you argue that this is a historical accident and an implementation detail?

Reply:
Are you saying we shouldn't mention the `match` opcode in the manual?